src/Acuminator/Acuminator.Analyzers/StaticAnalysis/DacExtensionDefaultAttribute/DacExtensionDefaultAttributeFix.cs

```diff
@@ -150,6 +138,20 @@ private async Task<Document> AddToAttributePersistingCheckNothing(Document docum
 			}
 
 			return document.WithSyntaxRoot(modifiedRoot);
+
+			AttributeArgumentSyntax getAttr()
```

Comment:
@saloponov please use .Net convention for method names for local methods too. Names should start with Uppercase letter. Also it's better to use a meaningful name without shortcut. 
For example `GetAttribute` is not much longer but has better readablilty and satisfies .Net conventions